Hillel/HW_9/Task_4.py

```diff
@@ -0,0 +1,11 @@
+result_link = 'https://www.youtube.com/watch?v=some_id&list=some_list&index=6&t=0s'
+
+result_dict = {}
+
+for char in result_link:
+  if char not in result_dict:
+    result_dict[char] = 1
+  else:
+    result_dict[char] += 1
```

Comment:
for char in result_link:
  if char not in result_dict:
    result_dict[char] = result_link.count(char)